conf/modules/peddy_check.config

```diff
@@ -17,6 +17,7 @@
 
 process {
     withName: '.*:PEDDY_CHECK:PEDDY' {
+        ext.when = { !(workflow.profile.contains("test")) || workflow.stubRun }
```

Comment:
I would use `.equals("test")`  and `test_one_sample` here instead. Better to be specific since we have a profile called test_full as well.